charts/k8ez/templates/deployment.yaml

```diff
@@ -31,13 +34,32 @@ spec:
         heritage: {{ .Release.Service }}
         version: {{ .Chart.Version }}
     spec:
-      {{if .Values.imagePullSecrets}}
-      {{- with .Values.imagePullSecrets }}
+{{ if .Values.imagePullSecrets }}
+{{- with .Values.imagePullSecrets }}
       imagePullSecrets:
 {{ toYaml . | nindent 8 }}
-      {{- end }}
-      {{- end }}
+{{- end }}
+{{- end }}
       serviceAccount: {{ .Release.Name }}
+      enableServiceLinks: {{ .Values.enableServiceLinks }}
+      securityContext:
+        fsGroup: {{ .Values.podSecurityContext.fsGroup }}
+      affinity:
+        podAntiAffinity:
```

Comment:
I'm gussing this makes them try to schedule on different nodes where possible?

Reply:
yep, it tells it to prefer different nodes for scheduling, but to ignore for actual execution.  Meaning they can still run on the same nodes if they get scheduled that way, but it prefers during scheduling not to if possible.